apps/mobile/apis/postKakaoLogin.ts

```
@@ -38,7 +38,8 @@ export const postKakaoLogin = async ({
 
     const newAccessToken = result.accessToken ?? result.access_token;
     const newRefreshToken = result.refreshToken ?? result.refresh_token ?? '';
-    const { hasExpense } = result;
+    const { isNewUser, hasExpense, homeOnboarding, categoryOnboarding, remindOnboarding } = result;
```

Biome (2.4.4)

[error] Shouldn't redeclare 'isNewUser'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)